src/config/configuration.ts

```diff
@@ -1,5 +1,6 @@
 export default () => ({
-  verseUrl: process.env.VERSE_URL ?? 'http://localhost:8545',
+  verseMasterNodeUrl: process.env.VERSE_MASTER_NODE_URL,
```

Comment:
Please add a fallback to the previous environment variable `VERSE_URL`

Reply:
Thanks!

I modified.
https://github.com/oasysgames/verse-proxy/pull/20/commits/9cf0db123549da7ff1c43b6cefb225fdab311f45